src/pkg/reg/adapter/aliacr/types.go

```diff
@@ -1,7 +1,6 @@
 package aliacr
 
 import "time"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380